### ISSUE TITLE
sass not compile

### ISSUE THREAD

**Issue by vipuser12** [glenn2223/vscode-live-sass-compiler#346]
### UNEXPECTED ERROR

**Machine & Versions**
| Item | Value |
|----------------------:|:-----------------------|
| VS Code | v1.86.2 |
| Platform | win32 x64 |
| Node | 18.17.1 (118) |
| Live Sass | 6.1.2 |
<details><summary>Installed Extensions</summary><div>
- vscode.css-language-features (1.0.0)<br/>- vscode.debug-auto-launch (1.0.0)<br/>- vscode.debug-server-ready (1.0.0)<br/>- vscode.emmet (1.0.0)<br/>- vscode.extension-editing (1.0.0)<br/>- vscode.git (1.0.0)<br/>- vscode.git-base (1.0.0)<br/>- vscode.github (0.0.1)<br/>- vscode.json-language-features (1.0.0)<br/>- vscode.merge-conflict (1.0.0)<br/>- vscode.npm (1.0.1)<br/>- vscode.typescript-language-features (1.0.0)<br/>- bradlc.vscode-tailwindcss (0.10.5)<br/>- esbenp.prettier-vscode (10.1.0)<br/>- formulahendry.code-runner (0.12.1)<br/>- glenn2223.live-sass (6.1.2)<br/>- ms-vscode.live-server (0.4.13)<br/>- ritwickdey.LiveServer (5.7.9)
</div></details>

**LOG**: 
```JSON
{
"NO LOG": "PLEASE SPECIFY YOUR ISSUE BELOW"
}
```
=======================

**Comment by glenn2223** [comment]
Sorry I'm going to need more information. What file? Has it ever compiled? Have you made the switch to the `@use` rule before it stopped? There might be a sass error, did the icon in the bottom left change when you save the file?

Have you regretted to the faq section https://github.com/glenn2223/vscode-live-sass-compiler/blob/master/docs/faqs.md

**Comment by vipuser12** [comment]
Thx glenn,
I solved the problem with auto compilation with live sass

But I can't configure tailwindcss well with sass

I installed tailwind but I noticed that Only one of them " tailwindCss or
SASS" works
Not together

Can you help me with config tailwindcss to work concurrently with sass?
, give me any instructions.

**Comment by glenn2223** [comment]
Depends on the context. Tailwind analysis your choice to create small CSS files whereas SASS just compiles into CSS. If you're creating content with tailwind classes compiling your CSS but having extra stylings coming from your SASS work then I don't see why they won't work side by side. If imagine you'd just have the 2 CSS links in your head elements

However, in the more likely case, you're building custom styles for tailwind via SASS. You probably don't want to and can refer to https://tailwindcss.com/docs/using-with-preprocessors

---

I'm closing this issue as you've resolved your problem but continue to comment if you like
